src/components/LanguageSelector.jsx

```diff
@@ -4,6 +4,12 @@ import MenuItem from 'material-ui/MenuItem';
 import SelectField from 'material-ui/SelectField';
 
 const LanguageSelector = ({ languages, onChange, selected }) => {
+  const styles = {
```

Comment:
I think we should move this, and the other inline styling, into a css file in the `/styles` folder.